packages/arb-bridge-eth/test/arbRollup.spec.ts

```diff
@@ -903,4 +952,46 @@ describe('ArbRollup', () => {
       node.connect(accounts[0]).addStaker(await accounts[1].getAddress())
     ).to.be.revertedWith('ALREADY_STAKED')
   })
+
+  it('should re-initialize', async function () {
```

Comment:
At first this sounded like calling initialize again on the same rollup. Maybe instead say "should initialize a fresh rollup chain"